subprojects/ide-native/src/main/java/org/gradle/ide/xcode/plugins/XcodePlugin.java

```diff
@@ -191,6 +199,57 @@ public void execute(SwiftLibraryPlugin plugin) {
                 configureXcodeForSwift(project, PBXTarget.ProductType.DYNAMIC_LIBRARY);
             }
         });
+
+        project.getPlugins().withType(XCTestConventionPlugin.class, new Action<XCTestConventionPlugin>() {
+            @Override
+            public void execute(XCTestConventionPlugin plugin) {
+                configureXcodeForXCTest(project, PBXTarget.ProductType.UNIT_TEST);
+            }
+        });
+    }
+
+    private void configureXcodeForXCTest(Project project, PBXTarget.ProductType productType) {
+        SwiftXCTestSuite component = project.getExtensions().getByType(SwiftXCTestSuite.class);
+        FileCollection sources = component.getSwiftSource();
+        xcode.getProject().getSources().from(sources);
+
+        // TODO - Reuse the logic from `swift-executable` or `swift-library` to determine the link task path
+        final CreateSwiftBundle bundleDebug = (CreateSwiftBundle) project.getTasks().getByName("bundleSwiftTest");
+        xcode.getProject().getSources().from(bundleDebug.getInformationFile());
+
+        // Sync the binary to the BUILT_PRODUCTS_DIR
+        final Sync syncTask = project.getTasks().create("sync" + GUtil.toCamelCase(component.getName()) + "BundleToXcodeBuiltProductDir", Sync.class, new Action<Sync>() {
+            @Override
+            public void execute(Sync task) {
+                task.dependsOn(bundleDebug);
+
+                final String buildProductsDir = System.getenv("BUILT_PRODUCTS_DIR");
```

Comment:
Shouldn't this just be something we have a convention for?  What's the typical value of this?